rdr_client/hpo_assigner.py

```diff
@@ -25,11 +24,13 @@ def main(client):
   num_no_change = 0
   num_updates = 0
   num_errors = 0
+
   with open(client.args.file) as csvfile:
     reader = csv.reader(csvfile)
     for line in reader:
       try:
         participant_id, hpo = [v.strip() for v in line]
+        pairing = client.args.pairing
```

Comment:
We should have validation that pairing is one of awardee, organization, and site here.

Reply:
done